src/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from './components/Calculator';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] One of the requirements is to use `React-testing-library` to simulate the user function. It would be better to use react testing library to make an operation as a user would do and add multiple cases like when the number is negative ...etc
- Screenshot of the requirements 
<img width="440" alt="Screenshot 2023-05-03 at 19 27 04" src="https://user-images.githubusercontent.com/61790587/236010503-961da615-741d-4433-a613-bc311046f09b.png">

Reply:
@youmari those operator tests are in the folder test kindly check it!

Reply:
@Yacoubou-seidou I saw them  those are unit tests but it would be better to test the calculator with react testing library for example by looking for button 1 and clicking then searching for button + and then clicking and click 3  expect to have 4 on screen this integration test like the user 
what you did is just test the function directly (unit tests)

Reply:
@Yacoubou-seidou here is an example of what I meant 
```javascript
import { render, fireEvent } from '@testing-library/react';
 it('should update the screen on button click', () => {
    const { getByTestId, getByText } = render(<Calculator />);
    fireEvent.click(getByText('2'));
    fireEvent.click(getByText('+'));
    fireEvent.click(getByText('3'));
    fireEvent.click(getByText('='));
    expect(getByTestId('screen')).toHaveTextContent('5');
  });
```

Reply:
@youmari, here are the project requirements
![requirement](https://user-images.githubusercontent.com/84572891/236331024-4d11b5ee-3acd-4034-9233-0f061e6415f0.png)

Reply:
@Yacoubou-seidou  here is in the requirements 
<img width="431" alt="Screenshot 2023-05-05 at 12 58 01" src="https://user-images.githubusercontent.com/61790587/236451850-4563d249-80e0-4b32-81e0-aa32068c8e9b.png">

Reply:
use React testing library to simulate user interaction